service/riotapi.py

```diff
@@ -2,17 +2,20 @@
 
 from errors.not_found_error import NotFoundError
 
+
 def get_puuid(game_tag, tag_line, api_key):
     # get the puuid
-    get_puuid_url = str.format("https://americas.api.riotgames.com/riot/account/v1/accounts/by-riot-id/{0}/{1}?api_key={2}",
-                               game_tag, tag_line, api_key)
+    get_puuid_url = str.format(
+        "https://americas.api.riotgames.com/riot/account/v1/accounts/by-riot-id/{0}/{1}?api_key={2}",
+        game_tag, tag_line, api_key)
     puuid_response_dict = get_response_json(get_puuid_url)
     return puuid_response_dict.get("puuid")
 
 
 def get_last_match(puuid, api_key):
-    get_match_url = str.format("https://americas.api.riotgames.com/lol/match/v5/matches/by-puuid/{0}/ids?start=0&count=20&api_key={1}", puuid,
-               api_key)
+    get_match_url = str.format(
+        "https://americas.api.riotgames.com/lol/match/v5/matches/by-puuid/{0}/ids?start=0&count=20&api_key={1}", puuid,
```

Comment:
count = 20 is technically incorrect here... never noticed that before. Also we have other tickets that signify the need for that to be a variable. Again, not your problem - just thinking out loud. Stuff like this is how we discover and create more "bug"/"improvement" tickets.